code/modules/mob/living/carbon/human/human_examine.dm

```diff
@@ -54,7 +54,11 @@
 				break
 	if(skip_jumpsuit && skip_face || HAS_TRAIT(src, TRAIT_NOEXAMINE)) //either obscured or on the nospecies list
 		msg += "!"    //omit the species when examining
-	else if(displayed_species == "Slime People") //snowflakey because Slime People are defined as a plural
+	else
+		var/species_name =  lowertext(displayed_species)
+		if(displayed_species == "Slime People") //snowflakey because Slime People are defined as a plural
+			species_name = "slime person"
+		msg += ", \a [height]<b><font color='[examine_color]'> [species_name]</font></b> with \a [physique] physique!"
 		msg += ", \a [height]<b><font color='[examine_color]'> slime person</font></b> with \a [physique] physique!"
 	else
 		msg += ", \a [height]<b><font color='[examine_color]'> [lowertext(displayed_species)]</font></b> with \a [physique] physique!" //Look I know it uses physique twice, lets not think about it too much ok?
```

Comment:
```suggestion
```
This can all go away I think

Reply:
fixed in the commit i posted at the exact same time as this review

Reply:
Lmao, timing